components/applications-service/pkg/storage/postgres/db.go

```diff
@@ -106,6 +106,7 @@ type service struct {
 	Channel             string    `db:"channel"`
 	FullPkgIdent        string    `db:"package_ident"`
 	PreviousHealth      string    `db:"previous_health"`
+	UpdateStrategy      string    `db:"update_strategy"`
```

Comment:
I suppose we could also use a PG ENUM here. But I know it's cumbersome to do all the translation... (we've got example code for this in authz-service, but using `database/sql`)

Reply:
What would it be the benefits of using an ENUM in postgres?

Reply:
The same as your storage layer's enum, I guess, just one layer down. Anyways, in authlahoma, we've stopped doing that, as it's cumbersome to deal with enums _on every layer_. 🙄